Add callout for licensing in kubernetes docs (#7322)
mattermost/docs: source/install/install-kubernetes.rst

    stringData:
      license: <LICENSE_FILE_CONTENTS>

  .. note::

>     An enterprise license is required for a seamless experience running a Mattermost installation with multiple server instances. If you will be deploying Mattermost without an enterprise license it is recommended to only use a single server instance. This can be done by adding ``Replicas:  1`` to the ``spec`` section in step 2.

Reviewer: ```suggestion
    A Mattermost Enterprise license is required for a seamless experience deploying Mattermost with multiple server instances. If you plan to deploy Mattermost without an Enterprise license, we recommend a single server deployment by adding ``Replicas:  1`` to the ``spec`` section in step 2.
```

>     For more information a running highly-available Mattermost deployment please refer to :doc:`this documentation </scale/high-availability-cluster-based-deployment>`.

Reviewer: ```suggestion
    See the :doc:`high availability </scale/high-availability-cluster-based-deployment>` documentation for details on running a highly-available Mattermost deployment.
```
